maro/simulator/scenarios/finance/common/commission.py

```diff
@@ -0,0 +1,120 @@
+from enum import Enum
+
+from maro.simulator.scenarios.finance.common.common import OrderDirection, two_decimal_price
+
+
+class CommissionType(Enum):
+    no_commission = 0
+    stamp_tax_commission = 1
```

Comment:
since there are both ``tax`` and ``commission`` in ``TradeResult``, they are 2 types of cost? But here the ``stamp_tax_commission`` belongs to ``commission`` ?